politeia_sync.go

```diff
@@ -62,11 +62,17 @@ func (p *Politeia) Sync() error {
 		}
 
 		log.Info("Politeia sync: update complete")
+		p.lastSyncedTimestamp = time.Now().Unix()
+		p.saveLastSyncedTimestamp(p.lastSyncedTimestamp)
```

Comment:
I dont think `p.lastSyncedTimestamp` is needed since it is not used outside this method.
```suggestion
		p.saveLastSyncedTimestamp(time.Now().Unix())
```